insights/parsers/hexdump.py

```diff
@@ -0,0 +1,50 @@
+"""
+HexDump - command ``/usr/bin/hexdump -C /dev/cpu_dma_latency``
+==============================================================
+
+This module provides the class ``HexDump`` which processes
+``/usr/bin/hexdump -C /dev/cpu_dma_latency`` command output.
+"""
+from insights import parser, CommandParser
+from insights.specs import Specs
+
+
+@parser(Specs.hexdump)
+class HexDump(CommandParser):
+    """
+    Class for parsing the output of `/usr/bin/hexdump -C /dev/cpu_dma_latency` command.
+
+    Typical output of is::
+
+        00000000  01 00 00 00                                       |....|
+        00000004
+
+    Attributes:
+        data(list): A string containing hexadecimal value on first line of the output.
+        hex_value (string): The hexadecimal value of force_latency.
+        dec_value (int): The decimal value of force_latency.
+
+    Examples:
+        >>> type(hex_dump)
+        <class 'insights.parsers.hexdump.HexDump'>
+        >>> hex_dump.data
+        ['00000000  01 00 00 00                                       |....|', '00000004']
+        >>> hex_dump.hex_value
+        '00 00 00 01'
+        >>> hex_dump.dec_value
+        1
+    """
+
+    def parse_content(self, content):
+        # The self.data returns the raw output of the command.
+        self.data = content
```

Comment:
`data` is not a good attribute for parsers anymore, since the name is a bit ambiguous and cannot indicate the real structure it stored.
If you want to add an attribute to return the `RAW` content, better rename it to `raw_content` or other.

Reply:
As discussed, I think it would be better to implement the parser more generically. The `hexdump` is a command to dump any kind of file to the `hex` format. In case the target file (`/dev/cpu_dma_latency` in this case) is different from the examples or polluted, and results in the `hexdump` doesn't output as expected in the `typical output`, the parser will fail. Please research a bit about the output of `hexdump` and refine the parser.

Reply:
Sure.

Reply:
Replaced `hexdump` with `od` command.